strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java

```diff
@@ -53,6 +55,8 @@ protected void configure(HttpSecurity http)
         http.sessionManagement()
             .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
             .and()
+            .exceptionHandling().authenticationEntryPoint(basicAuthenticationEntryPoint())
```

Comment:
@fuss86 I think that `basicAuthenticationEntryPoint` shouldn't be hard coded like this, because it should be provided with `Authenticator` somehow. For example we can add the target `user-agent` for `Authenticator`, and choose `EntryPoint` this way. Also we can have a default `Authenticator` to use for `EntryPoint` and this can be a `BasicAuthAutenticator`.